src/structs/index.md

```diff
@@ -5,26 +5,26 @@ languages. As long as you tell the Rust compiler to lay out structs "how C does
 it" (with `#[repr(C)]`) and have the right type declarations, everything *Just 
 Works™*. It's all bytes at the end of the day, anyway.
 
-Remember how I mentioned earlier that there's an easier way to compile and link
-everything? Well there is, it's called [gcc-rs][gcc-rs]. All you need to do is 
+Remember that there's an easier way to compile and link everything?
```

Comment:
I'm trying find where the book mentioned 'gcc-rs' before.
Because if it didn't, then the paragraph could jump right in with "There's an easy way to ..."

I was trying to find a way to keep the question, but without using "we"